src/java/fr/paris/lutece/plugins/suggest/web/SuggestApp.java

```diff
@@ -1186,29 +1186,12 @@ private void doVote( String strVote, int nIdSubmitSuggest, Plugin plugin, String
         // Increment vote
         int nScore;
 
-        if ( ( strVote != null ) && strVote.equals( "-2" ) )
+        if ( ( strVote != null ) && strVote.matches( ( "-2|-1|1|2" ) ) )
```

Comment:
This will compile the regex on each invocation. Maybe the compiled regex should be a static final class member ?

Reply:
Understood.